hpat/tests/test_series.py

```diff
@@ -1711,6 +1712,35 @@ def test_impl(S):
         S = pd.Series(np.random.ranf(m))
         self.assertEqual(hpat_func(S), test_impl(S))
 
+    def test_series_median_skipna_default1(self):
+        '''Verifies median implementation with default skipna=True argument on a series with NA values'''
+        def test_impl(S):
+            res = S.median()
+            print(res)
+            return res
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([2., 3., 5., np.nan, 5., 6., 7.])
+        self.assertEqual(hpat_func(S, ), test_impl(S))
+
+    def test_series_median_skipna_false1(self):
+        '''Verifies median implementation with skipna=False on a series with NA values'''
+        def test_impl(S):
+            res = S.median(skipna=False)
+            print(res)
```

Comment:
Is it debuggable `print`? If it's so please remove.

Reply:
Removed.